hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -20,6 +20,7 @@
 import static org.apache.hadoop.hbase.HConstants.DEFAULT_HBASE_SPLIT_COORDINATED_BY_ZK;
 import static org.apache.hadoop.hbase.HConstants.HBASE_MASTER_LOGCLEANER_PLUGINS;
 import static org.apache.hadoop.hbase.HConstants.HBASE_SPLIT_WAL_COORDINATED_BY_ZK;
+import static org.apache.hadoop.hbase.HConstants.MASTER_HOSTNAME_KEY;
```

Comment:
Unused import?

Reply:
Fixed.

Reply:
Actually it is needed here, I moved it to HConstants, so have to explicitly import now.